app/src/main/java/org/eu/droid_ng/wellbeing/lib/WellbeingService.kt

```diff
@@ -281,11 +317,52 @@ class WellbeingService(private val context: Context) {
 			}
 		}
 
+		setWellbeingAirplaneMode(enable &&
+				prefs.getBoolean("airplane_mode", false))
```

Comment:
save setting in field, don't reload every time

Reply:
I copied the code you did for `greyscale`, do you want me to also do this change for `greyscale`?

Reply:
oops, yes